### PR TITLE
Fix space removing after <inline><block/></inline> construction

### LINKED ISSUE #581: <a> with <code>
Hello!

When I minify piece of code like this:

`<p><a href="#"><code>Code word</code></a> and other words</p>`

I am getting this:

`<p><a href="#"><code>Code word</code></a>and other words</p>`

Is it possible to keep space after closing anchor tag? 

I use Gulp plugin, v.1.3.0, settings:

`{
    removeComments: true,
    collapseWhitespace: true
}`

### PR DESCRIPTION
**Pull request by igoradamenko**
This PR fixes #581.

### CODE REVIEW

File: src/htmlminifier.js [hunk 1]
```diff
@@ -978,10 +978,7 @@
             buffer.push('</' + tag + '>');
           }
           charsPrevTag = '/' + tag;
-          if (!inlineTextTags(tag)) {
```
Comment (alexlamsl) [inline src/htmlminifier.js:981]: You need to keep this for clearing `currentChars` - try changing this to `inlineTags` instead of removing the logic.

Comment (igoradamenko) [inline src/htmlminifier.js:981]: @alexlamsl yep, you are right; it works. I've added yet another commit

**Comment by alexlamsl** [comment]
@igoradamenko thanks for fixing this - can you squash the commits together? I will merge it right after :wink:

**Comment by igoradamenko** [comment]
@alexlamsl Done.

**Comment by alexlamsl** [comment]
:+1:

**Comment by igoradamenko** [comment]
@alexlamsl Could you also bump version to 1.3.2 for make fixes available in npm?

**Comment by alexlamsl** [comment]
@igoradamenko will do after merging a few more PRs :wink:

**Comment by igoradamenko** [comment]
@alexlamsl Okay, thanks!

**Pull request merged**